custom_components/plugwise-beta/config_flow.py

```diff
@@ -42,34 +68,117 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
+    def __init__(self):
+        """Initialize the Plugwise config flow."""
+        self.discovery_info = {}
+
+    async def _async_set_unique_id(self, id):
+        """Set the config entry's unique ID (based on the device's 4-digit PIN)."""
```

Comment:
Not sure where this came from, but we don't have PINs

Reply:
Copy-paste leftover :), solved.